docs/src/main/asciidoc/security.adoc

```diff
@@ -236,64 +265,76 @@ quarkus.http.auth.permission.bearer.policy=authenticated
 quarkus.http.auth.permission.bearer.auth-mechanism=bearer
 ----
 
-The value of the `auth-mechanism` property must match the authentication scheme supported by HttpAuthenticationMechanism such as `basic` or `bearer` or `form`, etc.
+Ensure that the value of the `auth-mechanism` property matches the authentication scheme supported by `HttpAuthenticationMechanism`, for example, `basic`, `bearer`, or `form`.
 
-== Proactive Authentication
+== Proactive authentication
 
-By default, Quarkus does what we call proactive authentication. This means that if an incoming request has a
-credential then that request will always be authenticated (even if the target page does not require authentication).
-
-See xref:security-built-in-authentication.adoc#proactive-authentication[Proactive Authentication] for more information.
+By default, Quarkus does proactive authentication, which means that all incoming requests with credentials are authenticated regardless of whether the target page requires authentication.
+For more information, see xref:security-built-in-authentication.adoc#proactive-authentication[Proactive authentication].
 
 == Authorization
 
-See xref:security-authorization.adoc[Security Authorization] for more information about Role Based Access Control and other authorization options.
+Quarkus also supports role-based access control (RBAC). 
+For more information about RBAC and other authorization options in Quarkus, see xref:security-authorization.adoc[Security authorization].
 
-== Customization and other useful tips
+== Quarkus Security customization
```

Comment:
@sberyozkin I wondered whether this section belongs higher up with the Security architecture information. What do you think?

Reply:
@michelle-purcell Sure, may be above `Combining authentication mechanisms` ?

Reply:
Moved :+1:  I also moved '**Authorization**' up with it too.